docs/excel/call-excel-apis-from-custom-function.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Call Excel APIs from a custom function
+description: Learn which Excel APIs you can call from your custom function. 
+ms.date: 02/06/2020
+localization_priority: Normal
+---
+
+# Call Excel APIs from a custom function
+
+[!include[Running custom functions in browser runtime note](../includes/excel-shared-runtime-preview-note.md)]
+
+Call Office.js Excel APIs from your custom functions to get range data and obtain more context for your calculations.
+
+Calling Office.js APIs through a custom function can be helpful when:
+
+- A custom function needs to get information from Excel before calculation. This information might include document properties, range formats, custom XML parts, a workbook name, or other Excel-specific information.
+- A custom function will set the cell's number format for the return values after calculation.
+
+[!include[Excel shared runtime note](../includes/note-requires-shared-runtime.md)]
+
+## Code sample
+
+To call into the Office.js APIs you first need a context. Use the `Excel.RequestContext` object to get a context. Then use the context to call the APIs you need in the workbook.
```

Comment:
Its a surprise that the instructions say call **Excel.RequestContext** because we normally recommend that people use **Excel.run** instead. If there's a reason for not using **Excel.run** in this scenario, I think we need to add an explanation of that here. If there is no reason, then why doesn't the article use **Excel.run**?